src/gtc/gtcpp/oir_to_gtcpp.py

```diff
@@ -54,7 +54,12 @@ def _extract_accessors(node: eve.Node) -> List[gtcpp.GTAccessor]:
     ndims = dict(
         node.iter_tree()
         .if_isinstance(gtcpp.AccessorRef)
-        .map(lambda accessor: (accessor.name, 3 + len(accessor.data_index)))
+        .map(
```

Comment:
Out of curiosity why are the temporaries excluded here?

Reply:
That makes refactoring rather tricky  if `GTAccessors` ever changes, e.g. in case it would be extended to supporting higher dimensionality, but I see the point.

Reply:
For sure it's not ideal. I figured a rewrite would tackle this at some point if more work is to be done on the backend. For now it's relatively stable so I wanted to keep changes relatively small.